quickjs/src/androidTest/java/com/squareup/quickjs/QuickJsSetTest.java

```diff
@@ -0,0 +1,1149 @@
+/*
```

Comment:
Copy of DuktapeSetTest, with a couple tests @Ignored for exception propagation issues (will do in follow-up)